### ISSUE TITLE
Kernel filtering not correctly replicated

### ISSUE THREAD

**Issue by TerrestrialEclipse** [pg020196/Neural-Network-Translator#144]
Keras and Pytorch apply kernels differently: 

- Keras cuts off the filter kernel at the edges
- Pytorch implicitly invokes zero padding to apply the whole filter kernel on the padded image

This could potentially be solved by encoding the intended kernel filtering behavior into the intermediate format. This also involves adapting the backends.

**Comment by pg020196** [comment]
Hi @TerrestrialEclipse,

thank you for pointing to this issue. In order to resolve this, I would agree to extend the intermediate format with an additional property for the kernel filtering behaviour, as you proposed in your initial post. 
This would result in simply setting the new property in the keras frontend and adjusting the backends accordingly.
